### ISSUE TITLE
Show the restore output when inline packages are used

### ISSUE THREAD

**Issue by filipw** [filipw/dotnet-script#193]
At the moment there is no feedback to the user that `dotnet restore` running - and this can be a slow/long operation, which can also fails.
We need to define a strategy of when do we show the user info that restore is running, and how it should be presented. 

This applies to both script and REPL.

**Comment by seesharper** [comment]
I agree that we should provide some kind of feedback, although we must be careful not to clutter the standard output here as that would cause trouble in piping scenarios. Would it be terrible wrong to write to standard error? Those are two different streams, right?

**Comment by seesharper** [comment]
`stderr` can be used for diagnostics so I guess it's a philosophical question whether `dotnet restore` output can be seen as diagnostics 😃

**Comment by seesharper** [comment]
Okay, so we already do logging to `stderr` Glad I had this conversation with  myself 😄 . 
So given that we should display anything in the console, should we always do it or maybe only if the restore exceeds a specified number of milliseconds?

**Comment by filipw** [comment]
It's OK to write it into `stderr`. 
I was thinking about something along these lines: always write a message when a restore is invoked. We could start with something simple i.e. restore started, restore finished (successfully/not successfully?) only. And if debug is enabled, redirect the restore process output too.
